### ISSUE TITLE
Windows 2025 TeXLive - LuaLaTeX Accessibility Failure in math

### ISSUE THREAD

**Issue by vinothkumars-cmd** [latex3/tagging-project#997]
Hi,

While using this below tex file in Windows 2025 texlive distribution it shows error "tagging-setup={mathml ... - Unknown". But the same document generates successful accessible PDF with mathml property in overleaf (2025 texlive setting).

Please help how to fix this issue in my windows 2025 distribution.

\documentclass{article}
\usepackage{luamml}

% Set up the tagging for MathML
\luamml[tagging-setup={math/setup=mathml-SE}]

\begin{document}

\title{Sample Document with MathML}
\author{Your Name}
\date{\today}
\maketitle

\section{Introduction}
This is a sample document demonstrating the use of the \texttt{luamml} package to create a tagged PDF with equations.

\section{Mathematical Equations}
Here is an inline equation: \( E = mc^2 \).

And here is a displayed equation:
\[
\int_0^\infty e^{-x^2} \, dx = \frac{\sqrt{\pi}}{2}.
\]

\end{document}

**Comment by u-fischer** [comment]
This can not work. Where did you get all this wrong commands from ???

Try with 

~~~~
\DocumentMetadata{tagging=on,tagging-setup={math/setup=mathml-SE}}
\documentclass{article}

\begin{document}

\title{Sample Document with MathML}
\author{Your Name}
\date{\today}
\maketitle

\section{Introduction}
This is a sample document demonstrating the use of the \texttt{luamml} package to create a tagged PDF with equations.

\section{Mathematical Equations}
Here is an inline equation: ( $E = mc^2$ ).

And here is a displayed equation:
\[
\int_0^\infty e^{-x^2} , dx = \frac{\sqrt{\pi}}{2}.
\]

\end{document}

~~~~

**Comment by vinothkumars-cmd** [comment]
[testing2.pdf](https://github.com/user-attachments/files/22764124/testing2.pdf)

Got this sample from one of the listed issue. But is there any better simple way to get accessbility math equation support. Your support/reply is very valuable.

**Comment by u-fischer** [comment]
Something is not up-to-date.  Update your texlive and use lualatex-dev.

**Comment by vinothkumars-cmd** [comment]
Hi Ulrike,

Thank you for the suggestion.

Tried texlive update but getting some error. Will check.

But the lualatex-dev run with non-stopmode gives below result with the given commands. 

<img width="1110" height="549" alt="Image" src="https://github.com/user-attachments/assets/937dcc3f-db54-47ae-86e3-8900d2fef5bc" />

Thanks but is there better way to tag the equation: because the reader ignore superscript/subscript etc...  Can we get further suggestion on how to achieve it.

<img width="1147" height="525" alt="Image" src="https://github.com/user-attachments/assets/f020acf7-5bb9-4213-a84a-5b91f350964d" />

**Comment by davidcarlisle** [comment]
Don't scroll past errors. After the first error saying the key is unknown, any pdf output and any tagging is at best a debugging aid and not usable.   If you update your system the math will be tagged with MathML not as a /Span.
